lib/index.js

```diff
@@ -363,6 +352,86 @@ module.exports = (bookshelf) => {
                 .where(wherePOUUID)
                 .del();
         },
+
+        bulkUpdate: function(rows, { transacting, returnInserted, columns } = {}) {
```

Comment:
Can you create a transaction if transacting is undefined/null?

Reply:
I think yes, no reason not to use transactions when it's not provided externally